### DISCUSSION TITLE
please help : how can i trigger a refresh for all the lookup in my app

### DISCUSSION THREAD

**Discussion by heleopless50** [serenity-is/Serenity#6983] [Help]
i have a sales app that one can have two companies and switch between them by switching the connection string and every thing is ok but there is a probleme that first company lookups are cashed and when i make switch to the other company it use the same lookups in [lookupeditor] how can i refresh all my lookups ??
thanks in advance

**Comment by VictorTomaili** [comment]
https://serenity.is/docs/tutorials/multi_tenancy/handling_lookup_scripts

**Reply by heleopless50** [reply]
i have read the documentation but my app is behaving differently
when i switch to another tenant i dont use tenant id to make the switch 
but the app connect to another database that has the same tables so 
in other words when i want to create a new tenant the app create a new database with new connection string 
and i have a master db to control the tenants and the users
the problem here that
one tenant may have two companies (two databases ) when he switches between them in realtime all services and connection are behaving correctly but the lookup scripts dont be updated 
so i want to trigger this when he press the button to connect to another database
like this 
**Q.reloadLookup("all lookups in the app")**
